Modelica/Fluid/Pipes.mo

```diff
@@ -940,7 +940,7 @@ This also allows for taking into account friction losses with respect to the act
 
             // Assumptions
             parameter Boolean allowFlowReversal=system.allowFlowReversal
-		  "= true, if flow reversal is enabled, otherwise restrict flow to design direction (states[1] -> states[n+1])"
+          "= true, if flow reversal is enabled, otherwise restrict flow to design direction (states[1] -> states[n+1])"
```

Comment:
There are several (many, in fact) cases in this package braking the new rule on indentation, e.g. the file's lines 3 or 901. So either fix them all (tedious) or let us first separate classes into particular files (cf. e.g. https://github.com/modelica/ModelicaStandardLibrary/pull/3433) and fix them afterwards.

Reply:
I believe that it should be done later - I just wanted to start by having the rules in place and fix the most obvious issues.

I agree that making all of the non-deliberate(?) indentation consistent will be tedious - I will see if it will be possible to automate (without changing the deliberate) - similarly as it is possible to move classes to separate files; but it will still take time to review.

Reply:
So simply resolve/close this conversation if desired.